fennel/test_lib/integration_client.py

```diff
@@ -11,8 +11,8 @@
     import pyarrow as pa
     from fennel_client_lib import RustClient  # type: ignore
     from fennel_dataset import lookup  # type: ignore
-except ImportError as e:
-    print(f"exception during import {e}")
+except ImportError:
+    pass
```

Comment:
Why not leave the print message?

Reply:
every time customers import fennel it is printing this exception, which looks wierd

Reply:
But are these import errors not fatal to using the int client?